src/features/Apiexplorer/RequestJSONBox/__tests__/RequestJsonBox.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { cleanup, render, screen, waitFor } from '@testing-library/react';
+import { TSocketEndpointNames } from '@site/src/configs/websocket/types';
+import useAuthContext from '@site/src/hooks/useAuthContext';
+import { IAuthContext } from '@site/src/contexts/auth/auth.context';
+import userEvent from '@testing-library/user-event';
+import RequestJSONBox from '..';
+
+jest.mock('@site/src/hooks/useAuthContext');
+
+const mockUseAuthContext = useAuthContext as jest.MockedFunction<() => Partial<IAuthContext>>;
+
+describe('RequestResponseRenderer', () => {
+  const mockProps = {
+    handleChange: jest.fn(),
+    request_example: '{"app_list": 1}',
+    name: 'app_list' as TSocketEndpointNames,
+    auth_required: 0,
+  };
+
+  beforeEach(() => {
+    mockUseAuthContext.mockImplementation(() => {
+      return {
+        is_logged_in: true,
+      };
+    });
+    render(<RequestJSONBox {...mockProps} />);
+  });
+
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('should render textarea', () => {
```

Comment:
please query the element with `textarea` and then assert on the label and placeholder.